main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+    2) win a game
+    3) move up the league table
+ */
+
+struct SoccerTeam
+{
+    class Manager
+    {
+        std::string name = "Pep Gaurdiola";
+        int Age = 48;
+        std::string birthPlace = "Spain";
+        int yearsActive = 15;
+        int numberOfTeamsManaged = 3;
+
+        void trainTeam(std::string nameOfCoach, bool deligateTrainingSession);
+        void signNewPlayers(std::string market, int age, std::string playerPosition);
+        void speakToThePress(bool attendPressConference = true);
+    };
+
+    // number of players in squad
+    int squadPlayers = 24;
+    // average age of players in squad
+    int averageAge = 28;
+    // number of total goals scored by the team
+    int totalGoalsScored = 60;
+    // number of matches in the season
+    int numOfMatchesToPlay = 38;
+    // league position
+    int leaguePosition = 4;
+
+    // train their skills
+    void trainingSession(Manager managerA);
+    // win a game
+    void winMatch(bool motivated);
+    // move up the league table
+    void changeLeaguePosition(int pointsAccquired);
+};
+
+/*
+Thing 4) Synth
+5 properties:
+    1) number of oscillators // int
+    2) amount of presets // int
+    3) number of filters // int
+    4) number of outputs // int
+    5) type of built-in effect // string
+3 things it can do:
+    1) output midi
+    2) filter out all frequences above 10khz
+    3) load different presets
+ */
+
+struct Synth
+{
+    // number of oscillators
+    int oscillators = 3;
+    // amount of presets
+    int presets = 150;
+    // number of filters
+    int filters = 3;
+    // number of outputs
+    int outputs = 2;
+    // type of built-in effect
+    std::string effectType = "delay";
+
+    // output midi
+    void outputMidi(int outputs);
+    // filter out all frequences above 10khz
+    void highCut10khz(bool filterSweepDown = true, bool adjustQFactor = true);
+    // load different presets
+    void loadPresets(bool loadAllPresets = false, std::string presetBankName = "Pads");
+};
+
+/*
+Thing 5) Seat
+5 properties:
+    1) outer material // string
+    2) inner material // string
+    3) how many riders it can seat // int
+    4) comfort rating // double
+    5) safety rating // double
+3 things it can do:
+    1) seat 2 riders at the same time
+    2) repel rain with its waterproof material
+    3) provide comfort for the rider
+ */
+
+struct Seat
+{
+    // outer material
+    std::string outerMaterial = "leather";
+    // inner material
+    std::string innerMaterial = "polyster";
+    // how many riders it can seat
+    int riders = 1;
+    // comfort rating
+    double comfortRating = 6.5;
+    // 5) safety rating 
+    double safetyRating = 9.5;
+
+    // seat riders
+    void seatRiders(int riders);
+    // repel rain with its waterproof material
+    void waterproof(bool applyWaterProofCoatToLeatherExterior = true);
+    // provide comfort for the rider
+    void comfort(int riders, double combinedWeightKG);
```

Comment:
none of these names contain the 'action' or verb.
change your function names to match the plain-english description more closely.
- [ ] revise